packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1669,33 +1669,6 @@ const provisioningStateSpecifiedForLROPatch = (patchOp, _opts, ctx) => {
     return errors;
 };
 
-const LROPostFinalStateViaProperty = (postOp, _opts, ctx) => {
```

Comment:
FYI I am not yet sure why these rules got deleted, probably some missed `rush build` command by someone else. I'll investigate and fix.

Reply:
OK this is definitely missing `rush build`. I'll have a PR for that in a moment.

Reply:
Fixed here:
- https://github.com/Azure/azure-openapi-validator/pull/540

Reply:
The problem is gone after rebasing on `main` and doing `rush build`.